packages/react/src/Button/Button.features.stories.tsx

```diff
@@ -63,6 +63,10 @@ export const TrailingCounter = () => {
 
 export const TrailingCounterWithNoText = () => <Button aria-label="Comments" leadingVisual={CommentIcon} count={3} />
 
+export const TrailingCounterWithHumanFormat = () => (
+  <Button aria-label="Comments" leadingVisual={CommentIcon} count={'3.2k'} />
```

Comment:
```suggestion
  <Button aria-label="Comments" leadingVisual={CommentIcon} count="3.2k" />
```

Small suggestion since this is a string and we don't need to the JSX expression syntax for this